CHARTER.md

```diff
@@ -0,0 +1,117 @@
+* A package format and manager to extend your control plane with:
+* The ability to orchestrate any kind of compute plane provider.
+* New ways to compose control plane APIs.
+* New integration with supporting services such as secret stores.
+* Tooling to create and test new Crossplane extensions.
+* A conformance program for Crossplane extensions.
+
+As a framework, Crossplane:
+
+* Avoids building features specific to any particular compute plane in its core.
+  Instead features pertaining to any particular cloud provider, project, or tool
+  are enabled through extension points.
+* Focuses on APIs. Crossplane provides and recommends “golden path” tools to
+  build conformant extensions but does not require them. Anything with
+  conformant API schema and behavior may be considered a Crossplane extension -
+  regardless of how it is built. 
+* Optimizes for “vertical integration”. That is, Crossplane extensions are
+  intended and designed to be used in conjunction with Crossplane and each
+  other.
+* Prioritizes enabling a “separation of concerns” between control plane curators
+  and consumers. Curators deploy, extend, and configure control planes, while
+  consumers use them to orchestrate compute planes.
+
+## What Crossplane Is Not
+
+Crossplane is not intended to be a “batteries included” control plane for any
+particular use case. Instead it is a framework that you can deploy, configure,
+and extend to power control planes for your own highly bespoke use cases, often
+without writing any code.
+
+The project is a neutral place for vendors and individuals to come together in
+enabling control planes. To that extent, it does not police Crossplane
+extensions. Any extension that passes the appropriate Crossplane conformance
+suite may consider itself to be “a Crossplane extension” - regardless of how it
+is built, how it is licensed, who maintains it, how it is governed, or where it
+is hosted. Conformance suites are typically concerned with behavior and (where
+applicable) API shape, not internal implementation details.
+
+Concrete examples of things that are not in scope for the project include (but
+are not limited to):
+
+* Providing or mandating abstractions around specific use cases such as
+  provisioning applications (e.g. a PaaS API) or provisioning Kubernetes
+  clusters. 
+* Providing or mandating a configuration language (for example, CUE). Crossplane
+  provides a declarative API that may be targeted by arbitrary configuration
+  languages, and extension points that may be powered by arbitrary languages.
+* Enabling the use of Crossplane extensions without using Crossplane and vice
+  versa; this is not supported and may result in undefined behavior.
+* Providing or mandating opinions around control plane hosting and tenancy
+  models.
+* Offering ‘in-tree’ support for particular compute planes - e.g. AWS. All
+  features specific to any specific compute plane provider are built as
+  extensions.
```

Comment:
```suggestion
  extensions such as providers.
```

nit to make that more clear.

Reply:
I'm kind of torn on this one - I was trying to avoid using terminology specific to the project.

Reply:
I started out wanting a more concrete definition for "extension", but found myself pleased with the generality by the time I got to this point :)

Reply:
Right. I'd love for this document to be specific enough to create some real bounds on project scope, but at the same time open ended enough that we don't need to update it very often as we tweak the features and architecture of the project.

Reply:
I'm going to leave 'providers' out here. I'm hoping that over time 'extensions' will be as meaningful as 'providers' (i.e. once it's more common to have extensions that are not providers).